doc/man3/RAND_load_file.pod

```diff
@@ -19,7 +19,11 @@ RAND_load_file, RAND_write_file, RAND_file_name - PRNG seed file
 RAND_load_file() reads a number of bytes from file B<filename> and
 adds them to the PRNG. If B<max_bytes> is nonnegative,
 up to B<max_bytes> are read;
-if B<max_bytes> is -1, the complete file is read.
+if B<max_bytes> is -1, the complete file is read (unless the file
```

Comment:
The part in parentheses is a bit confusing. Maybe something like this:

If B<max_bytes> is nonnegative, a regular file will be read completely. However, a file that is not a regular file will only be read 256 bytes at a time.